test/example-config-multiple-files-subtests-and-tasks/results.json

```diff
@@ -0,0 +1,62 @@
+{
+  "version": 3,
+  "status": "pass",
+  "tests": [
+    {
+      "name": "ExampleFirstTest.test_abc",
+      "status": "pass",
+      "test_code": "self.assertEqual(hello(13), (\"Hello, World!\", 13))",
+      "task_id": 1,
+      "output": "User output is captured! 13"
+    },
+    {
+      "name": "ExampleFirstTest.test_hello",
+      "status": "pass",
+      "test_code": "self.assertEqual(hello('Hi'), (\"Hello, World!\", 'Hi'))",
+      "task_id": 1,
+      "output": "User output is captured! Hi"
+    },
+    {
+      "name": "ExampleFirstOtherTest.test_dummy",
+      "status": "pass",
+      "test_code": "self.assertEqual(hello('Banana'), (\"Hello, World!\", \"Banana\"))",
+      "task_id": 2,
+      "output": "User output is captured! Banana"
+    },
+    {
+      "name": "ExampleFirstOtherTest.test_hello",
+      "status": "pass",
+      "test_code": "self.assertEqual(hello(42), (\"Hello, World!\", 42))",
+      "task_id": 2,
+      "output": "User output is captured! 42"
+    },
+    {
+      "name": "ExampleSecondTest.test_abc",
+      "status": "pass",
+      "test_code": "input_data = ['carrot', 'cucumber', 'grass', 'tree']\nresult_data = [(\"Hello, World!\", param) for param in input_data]\nnumber_of_variants = range(1, len(input_data) + 1)\n\nfor variant, param, result in zip(number_of_variants, input_data, result_data):\n    with self.subTest(f\"variation #{variant}\", param=param, result=result):\n        self.assertEqual(hello(param), result,\n                         msg=f'Expected: {result} but got something else instead.')",
```

Comment:
Is the whitespace correct here? I was expecting an indent after the `\n` before `msg=...`

Reply:
Nevermind, it's just GitHub rendering it poorly in the `Files Changed` view. It's correct in the main conversation 🙄